mfaliquot/application/updater.py

```diff
@@ -285,15 +285,16 @@ def postloop_finalize(self, terminated):
 
           _logger.info("Searching for merges...")
           merges = self.seqinfo.find_and_drop_merges()
-          if not merges:
-               _logger.info("No merges found")
-          else:
+          if merges:
                _logger.warning(f"Writing merges to {self.mergefile}")
-               for target, drops in merges:
-                    with open(self.mergefile, 'a') as f:
-                         f.write('{}_{}'.format('_'.join(str(d) for d in drops), target))
+               with open(self.mergefile, 'a') as f:
+                    for target, drops in merges:
+                         f.write('_'.join(str(seq) for seq in drops+(target,)) + '\n')
```

Comment:
The `target` needs to be the first in the list. Looking at what was there before it looks like it was also wrong. With your change this should read (or equivalent):
`f.write(target + '_' + '_'.join(str(seq) for seq in drops) + '\n')`